drivers/lora/sx1276.c

```diff
@@ -372,6 +376,12 @@ static int sx1276_lora_recv(struct device *dev, u8_t *data, u8_t size,
 	 */
 	memcpy(data, dev_data.rx_buf, dev_data.rx_len);
 
+	if (rssi != NULL)
```

Comment:
you need no add { } even on one line if statement, that's a style rule in Zephyr.

Reply:
Done :sweat_smile: